Tools/MediaDriverTools/GenDmyHex/CMakeLists.txt

```diff
@@ -22,6 +22,4 @@ cmake_minimum_required (VERSION 2.8)
 project(IntelGenDmyHexTool) 
 add_compile_options(-std=c++11)
 
-add_definitions(-DLINUX_)
```

Comment:
I think that all cmake level changes on Tools/MediaDriverTools/ level are fine. Don't remove them.

Reply:
Thanks, I've kept all the CMake changes to Tools/MediaDriverTools.